test/unittests/Database_Test.php

```diff
@@ -73,6 +73,44 @@ function testSetFakeData() {
                 )
 
             )
+        ); 
+
+        $DB->setFakeTableData(
```

Comment:
This should not be in the testSetFakeTableData test.
The new testDeleteWithIsNull should call setFakeTableData then `$DB->run("DROP TEMPORARY TABLE ConfigSettings")`